style.css

```diff
@@ -279,34 +296,333 @@ section .circle2 {
 }
 
 .recent-work .circle4 {
-  position: absolute;
+  position: sticky;
   width: 40.28px;
   height: 261.6px;
   left: 347.27px;
-  top: 2187.23px;
+
+  /* top: 2187.23px; */
   background: #edecff;
   filter: blur(25px);
   transform: matrix(-0.84, -0.54, -0.54, 0.84, 0, 0);
+  margin-top: -530px;
+  margin-bottom: 330px;
 }
 
 .recent-work .circle5 {
-  position: absolute;
+  position: sticky;
   width: 308px;
   height: 155px;
   left: -93px;
-  top: 3274px;
+
+  /* top: 3274px; */
   background: rgba(237, 236, 255, 0.3);
-  filter: blur(50px);
+  margin-top: -469px;
+  margin-bottom: 380px;
+  filter: blur(30px);
+  margin-left: -90px;
 }
 
 .recent-work .circle6 {
-  position: absolute;
-  width: 101.29px;
-  height: 377.04px;
+  position: sticky;
+  width: 81.29px;
+  height: 227.04px;
   left: 408.22px;
-  top: 3733px;
+
+  /* top: 3733px; */
   background: #edecff;
   filter: blur(30px);
   transform: matrix(-0.26, 0.97, 0.97, 0.26, 0, 0);
   border-radius: 50%;
+  margin-top: -480px;
+  margin-bottom: 250px;
+}
+
+.footer {
+  margin-top: 70px;
+  padding: 20px;
+  margin-bottom: 90px !important;
+}
+
+.footer .contact-head {
+  color: #9390ff;
+  font-size: 32px;
+  text-align: center;
+}
+
+.footer .contact-text {
+  font-size: 16px;
+  text-align: center;
+  color: #f5f5ff;
+  line-height: 24px;
+  margin-bottom: 30px;
+}
+
+.footer .form {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  width: 95%;
+}
+
+.footer .input {
+  height: 50px;
+  font-size: 15px;
+  margin-bottom: 15px;
+  padding-left: 10px;
+}
+
+.footer .form .textarea {
+  height: 130px;
+  padding-left: 10px;
+  margin-bottom: 50px;
 }
+
+.footer .form .submit-btn {
+  padding: 20px;
+  width: max-content;
+  margin: 0 auto;
+  background: #7d73fc;
+  color: #fff;
+  border: none;
+}
+
+.footer .divider {
+  width: 100%;
+}
+
+.footer .social-list {
+  margin-top: 30px;
+  text-align: center;
+}
+
+.footer .social-list ul {
+  display: inline-flex;
+}
+
+.footer .social-list ul li {
+  list-style-type: none;
+}
+
+.footer .next ul li a {
+  position: static;
+  width: 47px;
+  top: 4px;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 32px;
+  text-align: center;
+  color: #a2a3ae;
+  order: 1;
+  flex-grow: 0;
+  text-decoration: none;
+}
+
+.footer .email {
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  text-decoration: none;
+}
+
+.desktop-footer {
+  display: none;
+}

```

Comment:
- [ ] Your footer is not being shown on the desktop version due to this rule in your CSS. Remember that, if you do a `display: none` outside of a media query, you will need to do `display: block` (or inline or any you need) in order to show your element once you reach the breakpoint. Kindly fix this issue.

Reply:
- [ ] You have hidden the desktop footer in the mobile version. but it's still hidden in the desktop version. I would recommend adding `.desktop-footer {
  display: block;
}` inside the media query so your footer is shown on the desktop version.